RNTester/js/ProgressBarAndroidExample.android.js

```diff
@@ -16,33 +16,31 @@ var createReactClass = require('create-react-class');
 var RNTesterBlock = require('RNTesterBlock');
 var RNTesterPage = require('RNTesterPage');
 
-var MovingBar = createReactClass({
-  displayName: 'MovingBar',
-  _intervalID: (null: ?IntervalID),
+class MovingBar extends React.Component<{}> {
+  displayName = 'MovingBar';
```

Comment:
I think this should be:
```typescript
static displayName = 'MovingBar';
```

But since we have a babel plugin to automatically generate the `displayName` from components, we don't really need this. So, you should get rid of this line entirely.

Reply:
@RSNara thanks for helping me out , will fix pull request soon , learned a lot.